nixui/graphics/field_widgets.py

```diff
@@ -404,10 +372,9 @@ def validate_field(self, value):
         return value in self.choices
 
     def load_value(self, value):
-        if not self.validate_field(value):
-            value = ''
-        self.setCurrentIndex(self.choices.index(value))
-        self.loaded_value = value
+        if self.validate_field(value):
```

Comment:
It seems to only be changed for this specifically, should this be done for all?

Reply:
I believe so. If there isn't a valid choice for one-of, we don't set anything. This change isn't really part of the goal of this PR, but a bug I found.

Reply:
I think we should make another issue for this and have just a general bug squashing PR.

Reply:
Sure, but we should include this one here for ease rather than reverting IMO.

Reply:
Yep. We should keep this one.